migration/src/main/java/com/intershop/customization/migration/pfconfigurationfs/CfgResourceConverter.java

```diff
@@ -153,19 +196,32 @@ private ArrayList<String> migrateUserCfg(List<String> lines)
             }
             else
             {
-                    targetLine = this.prefix + ">" + line.trim();
-                    targetLines.add(targetLine);
+                targetLine = this.resourceType.getPrefix() + ">" + line.trim();
+                targetLines.add(targetLine);
             }
         }
         return targetLines;
     }
 
-    private ArrayList<String> migrateTransportCfg(List<String> lines) {
+    /**
+     * similar for configurations of #type# transport and application
+     * <p/>
+     * The ICM 7.10 configuration:<br/>
+     * <br/>
+     * gets converted to ICM11+:<br/>
+     * pfconfigurationfs>#type#>#UrlIdentifier#>#ParameterName# = #Value#<br/>
+     * pfconfigurationfs>#type#>rest>ExternalApplicationBaseURL = https://int-live-connect.roehm.com<br/>
```

Comment:
don't use customer names in examples
```suggestion
     * pfconfigurationfs>#type#>rest>ExternalApplicationBaseURL = https://int-live-connect.example.com<br/>
```